addons/binding/org.openhab.binding.samsungtv/ESH-INF/i18n/samsung_de.properties

```diff
@@ -0,0 +1,47 @@
+# binding
+binding.samsungtv.name = Samsung TV Binding
+binding.samsungtv.description = Dieses Binding integriert Samsung TV Ger�te. Durch dieses k�nnen Samsung TV Ger�te gesteuert werden.
```

Comment:
I think it wopuld sound better as one sentence.
Like: `Dieses Binding integriert Samsung TV Geräte, wodurch diese gesteuert werden können.`